.gitattributes

```diff
@@ -0,0 +1,81 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Files with these extensions are accepted source files inside the ReactOS
+# tree. They should have native line endings of the OS.
+*.acf text
+*.asm text
+*.c text
+*.cc text
+*.cmake text
+*.cpp text
+*.cxx text
+*.h text
+*.hpp text
+*.hxx text
+*.idl text
+*.inc text
+*.inl text
+*.klc text
+*.l text
+*.lds text
+*.md text
+*.rc text
+*.s text
+*.S text
+*.sfd text
+*.spec text
+*.svg text
+*.txt text
+*.TXT text
+*.y text
+
+# Files with these extensions end up in the built ReactOS system, so they
+# need to have CRLF line endings.
+*.bat text eol=crlf
+*.cmd text eol=crlf
+*.exp text eol=crlf
+*.hhc text eol=crlf
+*.hhp text eol=crlf
+*.ini text eol=crlf
+*.INI text eol=crlf
+*.inf text eol=crlf
+*.inf.tpl text eol=crlf
```

Comment:
This long extension makes me wonder whether the reactos.dff.in file in boot/bootdata/packages should also be CRLF or can be left as it is now.

Reply:
It has svn:eol-style native and build tools properly process it into a CRLF reactos.inf